packages/next-plugin-treat-example/src/components/Button.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import {button} from './Button.treat'
+
+export const Button = ({...otherProps}) => <button {...otherProps} className={button}/>;
```

Comment:
```suggestion
export const Button = (props) => <button {...props} className={button}/>;
```

Reply:
The example was adopted straight from the Gatsby counterpart.